services/digest_io.py

```
@@ -1,251 +1,370 @@
+        drafts.mkdir(exist_ok=True)
+        path = drafts / f"{date}.md"
+        path.write_text(markdown, encoding="utf-8")
+        logger.info(f"Saved markdown to {path}")
+        return path
+
+    def _validate_meta_kind(self, data: Dict[str, Any], expected_kind: str) -> None:
+        """Validate that data has the expected meta kind."""
+        meta = data.get("meta", {})
+        actual_kind = meta.get("kind")
+        if actual_kind != expected_kind:
+            raise ValueError(f"Expected {expected_kind} but got {actual_kind}")
```

Ruff (0.12.2)

Avoid specifying long messages outside the exception class

(TRY003)